daisy/compute/compute.go

```diff
@@ -267,6 +268,16 @@ func (c *client) DeleteInstance(project, zone, name string) error {
 	return c.i.operationsWait(project, zone, op.Name)
 }
 
+// DeprecateImage sets deprecation status on a GCE image.
+func (c *client) DeprecateImage(project, name string, deprecationstatus *compute.DeprecationStatus) error {
```

Comment:
Is it getting annoying having to wrap each API method? If so, we can talk about how to better approach the compute API.